R/pkg/R/stats.R

```diff
@@ -19,7 +19,8 @@
 
 setOldClass("jobj")
 
-#' crosstab
+#' @title SparkDataFrame statistic functions
```

Comment:
I don't see the build error - but the page title doesn't come up right. A screenshot is at https://www.dropbox.com/s/sc1mrd7upr6t7mp/Screenshot%202016-06-20%2021.25.57.png?dl=0

Also we seem to have some functions like covar_samp, covar_pop that don't have a description ?